assets/evaluation_results/squad_v2_gpt-4-0613_question_answering/spec.yaml

```diff
@@ -0,0 +1,33 @@
+type: evaluationresult
+name: squad_v2_gpt-4-0613_question_answering
+version: 2.10.05
+display_name: squad_v2_gpt-4-0613_question_answering
+description: gpt-4-0613 run for squad_v2 dataset
+dataset_family: squad_v2
+dataset_name: squad_v2
+
+model_name: gpt-4-0613
+model_version: "0613"
```

Comment:
model_version should be 4 as shown in Model Catalog registry..

Reply:
Hi, we have added the version for squadv2 dataset similar to the old results as in truthfulqa-generation dataset.

![image](https://github.com/Azure/azureml-assets/assets/38103118/fd9b32e8-994c-4495-bca5-ea411308a1ca)

can you please let me know if you suggest to update the version to 4?

Reply:
currently, we are trying to use the same version for gpt models as mentioned in truthfulqa_generation dataset.